propgeometry.txt

```diff
@@ -0,0 +1,36 @@
+----------------------------------------------------------------------

```

Comment:
Anche in questo caso sarebbe utile inserire questo file di testo nella cartella "TestCases"